packages/@divvi/mobile/src/public/components/Button.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import BaseButton, { BtnSizes, BtnTypes, ButtonProps, TextSizes } from 'src/components/Button'
```

Comment:
Moving forward we'll stop using absolute references as it requires more config and/or bundling that we don't end apps to need.

```suggestion
import BaseButton, { BtnSizes, BtnTypes, ButtonProps, TextSizes } from '../../components/Button'
```

Reply:
Fixed in e343247bd07daeafc3195aeb9a57662a5c00886f!